### PR TITLE
Feature/jjdd8fpo 29

### PR DESCRIPTION
**Pull request by DariuszCzajkowski**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: web/src/main/java/com/infoshareacademy/domain/api/AuthorJson.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.infoshareacademy.domain.api;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshareacademy/domain/api/AuthorJson.java]: po co ta adnotacja?

File: web/src/main/java/com/infoshareacademy/domain/api/EpochJson.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.infoshareacademy.domain.api;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+public class EpochJson {
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshareacademy/domain/api/EpochJson.java]: jw

File: web/src/main/java/com/infoshareacademy/domain/api/GenreJson.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.infoshareacademy.domain.api;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+public class GenreJson {
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshareacademy/domain/api/GenreJson.java]: jw

File: web/src/main/java/com/infoshareacademy/domain/api/KindJson.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package com.infoshareacademy.domain.api;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+
+public class KindJson {
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (rafalmisiak) [inline web/src/main/java/com/infoshareacademy/domain/api/KindJson.java]: jw

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**